x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.ts

```diff
@@ -167,7 +168,7 @@ export const getExecutionKPIAggregation = (filter?: IExecutionLogAggOptions['fil
               aggs: {
                 actionOutcomes: {
                   terms: {
-                    field: 'event.outcome',
+                    field: OUTCOME_FIELD,
```

Comment:
I think this should still be `event.outcome` since we're no longer changing it in the action event log docs